qa/os/src/test/java/org/elasticsearch/packaging/util/Docker.java

```diff
@@ -445,6 +448,14 @@ private static void verifyOssInstallation(Installation es) {
         ).forEach(executable -> assertPermissionsAndOwnership(es.bin(executable), p755));
 
         Stream.of("LICENSE.txt", "NOTICE.txt", "README.textile").forEach(doc -> assertPermissionsAndOwnership(es.home.resolve(doc), p644));
+
+        // These are installed to help users who are working with certificates.
+        Stream.of("zip", "unzip").forEach(cliTool -> {
+            // `which` isn't installed in our image, so instead just try to call the command.
+            final Shell.Result result = dockerShell.runIgnoreExitCode(cliTool + " -h");
```

Comment:
In bash you can also rely on `type -P <thefile>` if you don't have which to ensure the file is in $PATH:

e.g. on the Ubi image:

```
[elasticsearch@0f6edc7f150a ~]$ type -P zip
[elasticsearch@0f6edc7f150a ~]$ echo $?
1
[elasticsearch@0f6edc7f150a ~]$ type -P ls
/usr/bin/ls
[elasticsearch@0f6edc7f150a ~]$ echo $?
0
```

Reply:
`type -P` would work, but as it's a bash builtin, I'd have to run it through bash. That's a actually a bit cumbersome, because we're already running "bash -c 'docker blah blah commmand'". Instead, I'm now using `rpm` to check if the package is installed.